ios/InstagramFirebase/CommentsController.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  CommentsController.swift
+//  InstagramFirebase
+//
+//  Created by Brian Voong on 4/29/17.
+//  Copyright © 2017 Lets Build That App. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class CommentsController: UICollectionViewController, UICollectionViewDelegateFlowLayout, CommentInputAccessoryViewDelegate {
+    
+    var post: Post?
+    
+    let cellId = "cellId"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.title = "Comments"
+        
+        collectionView?.backgroundColor = .white
+        collectionView?.alwaysBounceVertical = true
+        collectionView?.keyboardDismissMode = .interactive
+        
+        collectionView?.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: -50, right: 0)
+        collectionView?.scrollIndicatorInsets = UIEdgeInsets(top: 0, left: 0, bottom: -50, right: 0)
+        
+        collectionView?.register(CommentCell.self, forCellWithReuseIdentifier: cellId)
```

Comment:
It'll be good if you write a separate function for the collectionView setup.
`private func setupCollectionView()`